src/android/WizPurchase.java

```diff
@@ -462,16 +462,23 @@ private void finishPurchase(JSONArray data) throws JSONException {
 	 * Get the list of purchases
 	 *
 	 **/
-	private JSONArray getPurchases() throws JSONException {
+	private JSONArray restorePurchasesAsPending() throws JSONException {
```

Comment:
Alternatively, pass the `List<Purchase>` as an argument to thie function. That way you don't need to return it at the end, and you can simply call this function `setPurchasesAsPending(purchases)`